plugins/out_cloudwatch_logs/cloudwatch_logs.c

```diff
@@ -345,11 +345,10 @@ static int cb_cloudwatch_init(struct flb_output_instance *ins,
     }
 
     /*
-     * Remove async flag from upstream
-     * CW output runs in sync mode; because the CW API currently requires
-     * PutLogEvents requests to a log stream to be made serially
+     * Keep cloudwatch networking on the async network stack.
+     * Opt into the FLB_OUTPUT_SYNCHRONOUS plugin option
      */
-    upstream->flags &= ~(FLB_IO_ASYNC);
+    // upstream->flags &= ~(FLB_IO_ASYNC);
```

Comment:
Let's just remove this commented code? And put the comment over the FLB_OUTPUT_SYNCHRONOUS invocation at teh bottom of the file?

Reply:
This sounds good!